src/hotspot/share/gc/parallel/parallelArguments.cpp

```diff
@@ -134,6 +134,15 @@ void ParallelArguments::initialize_heap_flags_and_sizes() {
     // Redo everything from the start
     initialize_heap_flags_and_sizes_one_pass();
   }
+
+  log_info(pagesize)("Alignments: "
+                     " space_align=" SIZE_FORMAT "%s"
+                     " gen_align=" SIZE_FORMAT "%s"
+                     " heap_align=" SIZE_FORMAT "%s",
+                     byte_size_in_exact_unit(SpaceAlignment), exact_unit_for_byte_size(SpaceAlignment),
+                     byte_size_in_exact_unit(GenAlignment), exact_unit_for_byte_size(GenAlignment),
+                     byte_size_in_exact_unit(HeapAlignment), exact_unit_for_byte_size(HeapAlignment)
+                     );
```

Comment:
As I wrote in the bug I think it makes sense to have a way to get this printed, and I think a better location might be to include it to the `gc+init` logging. Parallel doesn't have a specialized InitLogger yet, but you can create one and add it there. Take a look at `G1InitLogger` for insperation.

Reply:
Thanks for your comments.
Hava added a ParallelInitLogger. Please advice whether the logged information is sufficient.